SETUP.md

```diff
@@ -36,7 +36,7 @@ Next steps will assume you have a Github account and that you will get the code
 * Open a terminal and go on with the next steps in it.
 * Clone your forked repository: ```git clone git@github.com:YOURGITHUBNAME/android.git```.
 * Move to the project folder with ```cd android```.
-* Checkout the remote branch 'master' in your own local branch 'master': ```git checkout master remotes/origin/master```.
+* Checkout the remote branch 'master' in your own local branch 'master': ```git checkout master```.
```

Comment:
This will not work in modern git if you have multiple remotes, according to http://stackoverflow.com/questions/1783405/checkout-remote-git-branch#comment-40488052.
What is more, after cloning a repo, master is checked out by default. So this line can be removed altogether.